frontend/src/metabase/App.tsx

```diff
@@ -102,8 +102,10 @@ function App({
       return false;
     }
     if (IFRAMED) {
-      return EMBEDDED_ROUTES_WITH_NAVBAR.some(path =>
-        pathname.startsWith(path),
+      const isHomepage = pathname === "/";
+      return (
+        isHomepage ||
+        EMBEDDED_ROUTES_WITH_NAVBAR.some(path => pathname.startsWith(path))
```

Comment:
WDYT about changing `EMBEDDED_ROUTES_WITH_NAVBAR` to be regexes? This way you could add `/` there too